content/nginx-one/changelog.md

```diff
@@ -30,6 +30,13 @@ h2 {
 
 Stay up-to-date with what's new and improved in the F5 NGINX One Console.
 
+## September 16, 2025
+
+### IPv6 endpoints for NGINX Agent and NGINX Plus usage reporting
+
+Your instances running in dualstack or IPv6-only environments are now able to communicate with NGINX One APIs via IPv6.
```

Comment:
FYI, per our style guide, https://github.com/nginx/documentation/blob/main/documentation/style-guide.md, we avoid "latinisms". The industry seems to use "dual-stack". And I presume we're referring to NGINX One Console APIs from https://docs.nginx.com/nginx-one/api/api-reference-guide/

```suggestion
Your instances which run in dual-stack or IPv6-only environments can now communicate with NGINX One Console APIs through IPv6 addresses.
```